components/src/core/drawer/drawer-nav-group.tsx

```diff
@@ -0,0 +1,123 @@
+import React, { ReactNode, useState } from 'react';
+import { Subtitle } from '../typography';
+import { StyleSheet, View } from 'react-native';
+import { DrawerNavItem, NavItem, NestedNavItem } from './drawer-nav-item';
+import { inheritDrawerProps, NavGroupInheritableProps } from './inheritable-types';
+import { Divider } from 'react-native-elements';
+import Collapsible from 'react-native-collapsible';
+import * as Colors from '@pxblue/colors';
+import MatIcon from "react-native-vector-icons/MaterialIcons";
+
+export type DrawerNavGroupProps = {
+
+    // List of navigation items to render
+    items: NavItem[];
+
+    // Text to display in the group header
+    title?: string;
+
+    // Custom element, substitute for title
+    titleContent?: ReactNode;
+} & NavGroupInheritableProps;
+
+const styles = StyleSheet.create({
+    subtitle: {
+        paddingTop: 8,
+        paddingBottom: 8,
+        paddingLeft: 16,
+        height: 52,
+        lineHeight: 36,
+    },
+});
+
+// TODO: Can't this be replaced with a Set of itemIDs?
+function findID(item: NavItem | NestedNavItem, activeItem = ''): boolean {
+    // if leaf node, return
+    if (!item.items) {
+        return item.itemID === activeItem;
+    }
+
+    // else, loop through the branches
+    for (let i = 0; i < item.items.length; i++) {
+        if (findID(item.items[i], activeItem)) {
+            return true;
+        }
+    }
+
+    // no active items found, return false
+    return false;
+}
+
+export const DrawerNavGroup: React.FC<DrawerNavGroupProps> = (props) => {
+    const { title, titleContent, items } = props;
+
+    const getDrawerItemList = (item: NavItem | NestedNavItem, depth: number): ReactNode => {
+        const [expanded, setExpanded] = useState(findID(item, props.activeItem));
+
+        // Nested items inherit from the nestedDivider prop if item's divider is unset.
+        if (depth > 0 && item.divider === undefined) {
+            item.divider = props.nestedDivider as any;
```

Comment:
There's a type inconsistency here that's causing a bit of headache.  `divider` is a boolean, but in line 58 I check to see if it's `undefined`.  Line 59 throws a type error because `Type 'boolean' is not assignable to type 'undefined'.`

Reply:
Line 58 shouldn't be necessary because `divider` has a default value of true.  Going to update this.

Reply:
Maybe the best route is to make all inheritable boolean types undefined | boolean.

Reply:
yeah, divider should be an optional prop, so it should be able to take undefined.